src/NuGet.Clients/NuGet.VisualStudio/Extensibility/IVsPathContextProvider2.cs

```diff
@@ -42,5 +44,20 @@ public interface IVsPathContextProvider2 : IVsPathContextProvider
         /// <code>InvalidOperationException</code> when it fails to create a context and return appropriate error message.
         /// </throws>
         bool TryCreateSolutionContext(string solutionDirectory, out IVsPathContext2 context);
+
+        /// <summary>
+        /// Attempts to create an instance of <see cref="IVsPathContext"/> containing only the user wide and machine wide configurations.
+        /// If a solution is loaded, note that the values in the path context might not be the actual effective values for the solution.
+        /// If a customer has overriden the `globalPackagesFolder` key or cleared the `fallbackPackageFolders`, these values will be incorrect.
+        /// It is important to keep this scenario in mind when working with this path. To predict differences you can call this in combination with <see cref="TryCreateSolutionContext(out IVsPathContext2)"/>.
+        /// </summary>
+        /// <returns>
+        /// <code>True</code> if operation has succeeded and context was created.
+        /// <code>False</code> otherwise.
+        /// </returns>
+        /// <throws>
+        /// <code>InvalidOperationException</code> when it fails to create a context and return appropriate error message.
+        /// </throws>
+        bool TryCreateUserWideContext(out IVsPathContext vsPathContext);
```

Comment:
When I first saw the name TryCreateUserWideContext, I was thinking that "UserWide" meant that it only contained the user-specific settings and not the machine-wide settings. Is this a term that consumers will understand because it's used in other NuGet APIs?

If not, maybe it needs a better name? I don't have any spectacular ideas, but maybe something like TryCreateSolutionlessContext or TryCreateNoSolutionContext?

Reply:
It's something used in our docs.
I can't with high confidence claim that it'll obvious to customers. 

Others thoughts?

Reply:
I'd vote for something with "no solution" in the name. I like `TryCreateNoSolutionContext`.

Reply:
done.